client/App.jsx

```diff
@@ -19,11 +19,7 @@ class App extends React.Component {
 
   componentDidMount() {
     axios
-      .get("/mainbody", {
-        params: {
-          _id: proxyId,
-        },
-      })
+      .get(`/mainbody/${proxyId}`)
```

Comment:
What was the point of this change?

Reply:
I wanted the id in the ending not the parameters, and the whole object was messed up anyways. I didn't understand it. Now I do.